### ISSUE TITLE
Unable to use script from CDN

### ISSUE THREAD

**Issue by ad-git** [i18next/react-i18next#588]
I used in the past react-intl cdn without problem. Now i would like to try react-i18next, but when i add it in browser, from the CDN, i got this error

"TypeError: "__esModule" is read-only"

Thanks for your time.

**Comment by jamuhl** [comment]
how do you include it?

**Comment by ad-git** [comment]
I have included it on browser, in this way:

```
<!DOCTYPE html>
<html>
    <head>
        <meta charset="utf-8">
        <script>
            window.process = {
                env: {
                    NODE_ENV: 'production'
                }
            };
        </script>
        <script crossorigin src="https://unpkg.com/react@16/umd/react.production.min.js"></script>
        <script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.production.min.js"></script>
        <script src="https://unpkg.com/react-i18next/react-i18next.js"></script>  
        <script src="https://unpkg.com/react-i18next/react-i18next.min.js"></script>  
    </head>
    <body></body>
</html>
```

But suddenly i get this error: "TypeError: "__esModule" is read-only" 
You can try it at this  link: https://codepen.io/anon/pen/dgrKdN

Thanks for helping me

**Comment by jamuhl** [comment]
yes getting the error

`Cannot assign to read only property '__esModule' of object '#<Object>'`

not sure what it means or what to make out of the responses given by a fast google search. Not sure might be something with the rollbar build.

Would be great someone can look into this - currently not my personal highest prio.

**Comment by jamuhl** [comment]
Ok found it: solved in react-i18next@8.3.5

you can access it on window.ReactI18next (changed the name to be more in line with react)

**Comment by jamuhl** [comment]
If you like this module don’t forget to star this repo. Make a tweet, share the word or have a look at our https://locize.com to support the devs of this project -> there are many ways to help this project :pray:
